classes/blocks/class-sharebuttons.php

```diff
@@ -0,0 +1,75 @@
+<?php
+/**
+ * Hubspot Forms block class
+ *
+ * @package P4GBKS
+ * @since 0.1
+ */
+
+namespace P4GBKS\Blocks;
+
+/**
+ * Class ShareButtons
+ * Registers the ShareButtons block.
+ *
+ * @package P4BKS
+ * @since 0.1
+ */
+class ShareButtons extends Base_Block {
+
+	/**
+	 * Block name.
+	 *
+	 * @const string BLOCK_NAME.
+	 */
+	const BLOCK_NAME = 'share-buttons';
+
+	/**
+	 * ShareButtons constructor.
+	 */
+	public function __construct() {
+		add_action( 'init', [ $this, 'register_sharebuttons_block' ] );
+	}
+
+/**
+	 * Block attributes.
+	 *
+	 * @var array $attributes Block attributes definition.
+	 */
+	private static $attributes = [
+		'social_accounts'			=> [
+			'type' => 'object'
+		],
+		'social_params' 			=> [
+			'type' => 'object'
+		],
+		'version' 						=> [
+			'type' => 'integer',
+		],
+	];
+
+	/**
+	 * Register ShareButtons block.
+	 */
+	public function register_sharebuttons_block() {
+		register_block_type(
+			self::get_full_block_name(),
+			[
+				'render_callback' => [ $this, 'front_end_rendered_fallback' ],
```

Comment:
This fallback is not needed here because it's the first version of the block.

Reply:
Oh sorry, I missed to remove it! Thanks for pointed it this